### DISCUSSION TITLE
Перестали работать instagram и ntc.party

### DISCUSSION THREAD

**Discussion by tunivi** [xtrime-ru/antizapret-vpn-docker#106] [Q&A]
С сегодняшнего дня перестали работать instagram и ntc.party через антизапрет. При выполнении команды `nslookup instagram.com` и `nslookup ntc.party` выдаются их реальные IP, а не адреса из подсети 10.224
Пока временно прописал домены в include-hosts-custom.txt
В чём дело? Инста и ntc пропали из реестра заблокированных ресурсов Роскомнадзора?

UPD: прописывание ntc.party в include-hosts-custom.txt ничего не дало. По прежнему резолвится реальный IP. Инста заработала.

**Comment by xtrime-ru** [comment]
После обновления конфигов нужно перезапустить контейнер. Так же надо проверить правила в exclude.
